src/api/CallSP.ts

```diff
@@ -9,10 +9,17 @@
 *                                                                                 *
 */
 
-import { ConnectionString, IDB2Session, IDB2Response, IDB2Parameter, SessionValidator,
-    DB2Constants, DB2_PARM_INOUT, DB2_PARM_OUTPUT, DB2Error } from "../";
+// import { ConnectionString, IDB2Session, IDB2Response, IDB2Parameter, SessionValidator,
```

Comment:
Why is this commented out? Should it be removed entirely?

Reply:
done! 😋